ext/native-decls/CancelEvent.md

```diff
@@ -10,3 +10,9 @@ void CANCEL_EVENT();
 
 Cancels the currently executing event.
 
+## Examples
+```lua
+AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
+            CancelEvent()
```

Comment:
Hey, maybe I'm nitpicking, but could you please [indent](https://en.wikipedia.org/wiki/Indentation_style) this to the left using `SHIFT + Tab`?

i.e.
```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
    CancelEvent()
end)
```